### PR TITLE
Add CI/CD docs

### PR DESCRIPTION
**Pull request by msrb**

### CODE REVIEW

File: docs/README-cicd.md [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+
+### Step 1: Create a repository in quay.io Docker registry
+ 
+If you want to build Docker images and push them to quay.io registry (we push to quay.io, not to dockerhub),
+then you will need to whitelist the name of your Docker repository there.
+    
+Simply go to <internal GitLab URL>/housekeeping and follow instructions on how to whitelist your new repository.
+If you don't know the GitLab URL, ask you peers ;)
+    
+You will like want to whitelist two repositories:
```
Comment (tisnik) [inline docs/README-cicd.md]: will like want?

File: docs/README-cicd.md [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+Note `hash: none` has a special meaning. If the value is `none`,
+it means that your service won't be deployed to production. Only to staging environment.
+
+Create a config file for your service and open a pull request
+in [openshiftio/saas-analytics](https://github.com/openshiftio/saas-analytics) repository.
+
+
+### Step 6: Configure webhooks in your GitHub repository
+
+Can't believe you made it this far :) Don't worry, you're almost there!
```
Comment (tisnik) [inline docs/README-cicd.md:209]: I do :)

**Review by tisnik — APPROVED** [review]
Very good! And very important document as well @msrb

File: docs/README-cicd.md (hunk 1 above)
Comment (msrb) [reply docs/README-cicd.md]: s/like/likely/g :)

**Pull request merged**